pages/table/generate-data.ts

```diff
@@ -110,3 +110,338 @@ export function generateItems(count = 4000) {
     return value;
   });
 }
+
+export const generated50: Instance[] = [
```

Comment:
This is generated with `generateItems(50)`. We do however keep the IDs uniquely generated, that is because when combining the statically generated items with newly generated ones (e.g. when imitating items addition) the IDs were matching, causing duplicate keys React error.